pkg/eventservice/event_scanner_test.go

```diff
@@ -43,7 +43,11 @@ func makeDispatcherReady(disp *dispatcherStat) {
 }
 
 func (m *mockMounter) DecodeToChunk(rawKV *common.RawKVEntry, tableInfo *common.TableInfo, chk *chunk.Chunk) (int, *integrity.Checksum, error) {
-	return 0, nil, nil
+	if rawKV.IsUpdate() {
+		return 2, nil, nil
+	} else {
```

Comment:
remove the else, it's redundant

Reply:
it's necessary for some unit test cases so I add this.